src/components/NetworkAssetSelectInput/NetworkAssetSelectInput.tsx

```diff
@@ -377,12 +441,21 @@ const NetworkAssetSelectInput = ({
         (item: any) => item.chain === supportedChain.chainId
       );
       return smartWalletBalanceByChain?.length && balanceByChain.length
-        ? formatAmountDisplay(String(balanceByChain[0].total), '$')
+        ? ` · ${formatAmountDisplay(String(balanceByChain[0].total), '$')}`
         : '$0';
-    } else if (accType === DestinationWalletEnum.Key && label === 'From') {
-      return '$0';
+    } else if (
+      accType === DestinationWalletEnum.Key &&
+      label === 'From' &&
+      keybasedWalletBalanceByChain?.length
+    ) {
+      let balanceByChain = keybasedWalletBalanceByChain.filter(
+        (item: any) => item.chain === supportedChain.chainId
```

Comment:
Is there a way to avoid using `any`? Does `keybasedWalletBalanceByChain` have a type or something? Isn't to important in this case, but better to avoid it where you can. Maybe change this to `item?.chain` since we're using `any`